src/components/events/event-card.tsx

```
@@ -0,0 +1,36 @@
+'use client';
+
+import { Card, CardHeader, CardContent, CardDescription } from '@/components/ui/card';
```

ESLint

[error] 'CardDescription' is defined but never used.

(no-unused-vars)